bundle/Templating/Twig/Extension/NetgenTagsRuntime.php

```diff
@@ -95,4 +95,21 @@ public function getContentTypeName($contentType)
 
         return $contentType->getName();
     }
+
+    public function getParentTag($tag)
```

Comment:
You can typehint Tag for the argument. With that, you can also remove the `$tag instanceof Tag` check.

Reply:
Done in https://github.com/netgen/TagsBundle/commit/f4a8ea03a3e319d60e3f799889a2fd0b12598249